src/renderer/component/address/view.jsx

```diff
@@ -16,37 +16,35 @@ export default class Address extends React.PureComponent {
   }
 
   render() {
-    const { address, showCopyButton, doShowSnackBar } = this.props;
+    const { address, doShowSnackBar } = this.props;
 
     return (
-      <div className="form-field form-field--address">
-        <input
-          className={classnames('input-copyable', {
-            'input-copyable--with-copy-btn': showCopyButton,
-          })}
-          type="text"
-          ref={input => {
-            this._inputElem = input;
-          }}
-          onFocus={() => {
-            this._inputElem.select();
-          }}
-          readOnly="readonly"
-          value={address || ''}
-        />
-        {showCopyButton && (
-          <span className="header__item">
+      <FormField
+        render={() => (
```

Comment:
Using render props makes it really simple add additional items to the form field like this button. Instead of having a huge `FormField` element that can do everything, allowing the user of the component to do whatever they want can be  really nice.